File: 1-Patron/php/cliente.php
<?php 
 class Cliente { 
    private $nombre;
    private $apellido;
    private $edad;
    private $correo;
    private $contraseña;



    public function __construct($nombre,$apellido,$edad,$correo,$contraseña) {
    	$this->$nombre = $nombre;
    	$this->$apellido = $apellido;
    	$this->$edad = $edad;
    	$this->$correo = $correo;
    	$this->$contraseña = $contraseña;
    }

    public function getNombre(){
        return $this->nombre;
    }

    public function setNombre($nombre){
        $this->nombre = $nombre;
    }

    public function getApellido(){
        return $this->apellido;
    }

    public function setApellido($apellido){
        $this->apellido = $apellido;
    }
    public function getEdad(){
        return $this->edad;
    }

    public function setEdad($edad){
        $this->edad = $edad;
    }
    public function getCorreo(){
        return $this->correo;
    }

    public function setNombre($correo){
        $this->correo = $correo;
    }

    public function getContra(){
        return $this->contraseña;
    }

    public function setContra($contraseña){
        $this->contraseña = $contraseña;
    }
 }
?>

File: 1-Patron/Java/Cliente.java
package masterwholeproxy;

public class Cliente {
    private String nombre;
    private String apellido;
    private int edad;
    private String correo;
    private String contrasena;
    
    public Cliente(String nombre, String apellido, int edad, String correo,String contrasena ){
        this.nombre = nombre;
        this.apellido = apellido;
        this.edad = edad;
        this.correo = correo; 
        this.contrasena = contrasena; 
    }
    public String getNombre(){
        return this.nombre;
    }
  
    public String getApellido(){
        return this.apellido;
    }
    
    public int getEdad(){
        return this.edad;
    }
    
    public String getCorreo(){
       return this.correo;
    }
    public String getContrasena(){
        return this.contrasena;
    }
    public void setNombre(String nombre){
        this.nombre = nombre;
    }
    public void setApellido(String apellido){
        this.apellido = apellido;
    }
    public void setEdad(int edad){
        this.edad = edad; 
    }
    public void setCorreo(String correo){
        this.correo = correo;
    }
    public void setContrasena(String contrasena){
        this.contrasena = contrasena;
    }
    
}
  